samples/chat/README.md

```diff
@@ -24,7 +24,7 @@ Please refer to the root level [README](../../README.md#requirements) for prereq
     | .NET oo-proc | `cd samples/chat/csharp-ooproc && func start` |
     | Node.js | `cd samples/chat/nodejs && npm install && dotnet build --output bin && npm run build && npm run start` |
     | PowerShell | `cd samples/chat/powershell && dotnet build --output bin && func start` |
-    | Python | `cd samples/chat/python && dotnet build --output bin && func start` |
```

Comment:
missed deletion of extensions.csproj?

Reply:
It is deleted. Not sure why it doesn't show up clearly in the files changed section, but you can see it's not present in the files in this branch

Reply:
I see, its showing as moved to Kusto